frontend/packages/dev-console/src/components/pipelines/pipeline-builder/PipelineBuilderPage.scss

```diff
@@ -1,4 +1,5 @@
 .odc-pipeline-builder-page {
   height: 100%;
   position: relative;
+  overflow-x: hidden;
```

Comment:
What is this getting around? Horizontal overflow shouldn't really ever need to be explicitly mentioned if I did my job with the layout 😄

Reply:
@andrewballantyne I added this to avoid
![Peek 2020-05-19 18-48](https://user-images.githubusercontent.com/20724543/82331097-487cba80-9a01-11ea-914b-52390921de6a.gif)

Reply:
Interesting. I'll have to look to reproduce that. Did you zoom in? The bubble looks cut off.

Reply:
Yes, see if you can reproduce it. No I didn't zoom in.

Reply:
I cannot reproduce this. 🤔 You look to be using Chrome... is this on Linux? What version of Chrome?

Reply:
Yes I am using Chrome. Yup on linux.
@andrewballantyne just to be sure you tried to reproduce this after removing the overflow property, right? :sweat_smile: . If yes then it might be because of my display settings probably.

Reply:
Haha yes I did. I also tried several zoom features to try and reproduce the unbalance of the node. I think there is definitely something happening here with an upscale to your UI. I think it would be useful for us to determine what that is... even the icons for the "+" buttons look off.

Reply:
Yeah then its because of my display settings, I keep my scaling factor to 1.25 :P.

Reply:
So even with scaling factor set to 1 is still this :thinking: 
![Peek 2020-05-20 01-39](https://user-images.githubusercontent.com/20724543/82373235-ea6cc900-9a3a-11ea-955c-cdae63f7cfa9.gif)

Reply:
Interesting. Maybe this is just another reason for us to convert the node to SVG (from a foreignObject div usage)